dist/api_data.py

```diff
@@ -1338,7 +1338,7 @@ def __ge__(self, other):
     Config('bound', '', r'''
         configures which bound is being operated on. It takes one of two values, "lower" or "upper".
         ''',
-        choices=['lower','upper']),
+        choices=['lower','upper'])
```

Comment:
np: All the above configs I see maintained the ending comma

Reply:
d38d449d3 😤